moveit_ros/hybrid_planning/local_planner/local_planner_component/src/local_planner_component.cpp

```diff
@@ -277,12 +277,19 @@ void LocalPlannerComponent::executeIteration()
       *local_planner_feedback_ = local_constraint_solver_instance_->solve(
           local_trajectory, local_planning_goal_handle_->get_goal(), local_solution);
 
-      // Feedback is only send when the hybrid planning architecture should react to a discrete event
+      // Feedback is only sent when the hybrid planning architecture should react to a discrete event
       if (!local_planner_feedback_->feedback.empty())
```

Comment:
This looks like a pretty broad assumption. Should the feedback be handled in any way?